### ISSUE TITLE
Animation's `ComputeFromValue` is not working correctly

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#11896]
### Current behavior

The `Expander` style in WinUI has the following in `CheckedPointerOver` `VisualState`:

```xaml
  <contract7NotPresent:DoubleAnimation
      Storyboard.TargetName="ExpandCollapseChevronRotateTransform"
      Storyboard.TargetProperty="Angle"
      To="180.0" Duration="0:0:0.1"/>
```

However, the same is in the `Checked` `VisualState` to accomodate for keyboard navigation. When the animation finishes, the final value is kept, so subsequent pointer hover should not cause further animation (as the effective `From` value is already 180.0). However, in case of Uno, the chevron snaps to angle 0 and rotates quickly. 

### Expected behavior

No animation should occur on poitner over.

### Workaround

Our current workaround in `Expander_themeresources.xaml` was to replace all ocurrences of `Duration="0:0:0.1"` by `Duration="0:0:0"`. Revert this workaround to see the broken behavior.

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia, Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### IDE

Visual Studio 2022

**Comment by MartinZikmund** [comment]
@dr1rrb assigned this one to you, maybe it was reported in some other form before

**Comment by jeromelaban** [comment]
@MartinZikmund nice. Is it related to the expander chevron ?

**Comment by Xiaoy312** [comment]
looks like a duplicate of #631
would be addressed by: #11865
